provider-utils/awscloudformation/cloudformation-templates/vod-helpers/InputTriggerLambda.template

```diff
@@ -106,6 +106,13 @@ Resources:
                   - iam:PassRole
                 Resource:
                   - !GetAtt MediaConvertRole.Arn
+              - Effect: Allow
+                Action:
+                  - s3:DeleteObject
+                  - s3:ListBucket
+                Resource:
+                  - !Sub "${pOutputS3Arn}/*" 
+                  - !Ref pOutputS3Arn
```

Comment:
Should we remove these permissions if we aren't adding the delete functionality until later?